packages/api-server/src/app.ts

```diff
@@ -1,13 +1,9 @@
-import express from 'express'
-import type { Response, Request, Application } from 'express'
-import morgan from 'morgan'
+import Fastify, { FastifyInstance } from 'fastify'
 
-// Base express app, with common config
-const createApp = (): Application => {
-  const app = express()
-
-  // Add common middleware
-  app.use(morgan<Request, Response>('dev'))
+export const createApp = (): FastifyInstance => {
+  const app = Fastify({
+    logger: { prettyPrint: process.env.NODE_ENV !== 'production' },
```

Comment:
Isn't prettyPrint deprecated? https://github.com/pinojs/pino/pull/1203/files

Reply:
Yes, you are right it has. We'll check and see how we want to format the log output.

Reply:
Turn out that Fastify is still using Pino v6:

https://github.com/fastify/fastify/blob/6a0403319148163fe6f826aba16e33e46fdb88d4/package.json#L184

So, using prettyPrinting here would be ok -- especially if one for Development logging.

Or - do we hook in the Redwood logger created and configured in someone's app - similar to how the GraphQLServer package can log?

Maybe. Want to look into that.

Reply:
Fastify works with newer pino too, if we want to provide explicit dependency version.

Reply:
@kibertoad For this PR so we can merge, let's use Pino 6 with some prettifying in dev only -- and in a subsequent PR I am going to look into using latest Pino or a custom simple prettifier.